content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -62,23 +63,23 @@ To connect a MetaMask wallet to the bridge, go to the [EVM Sidechain bridge](htt
 
 ![Connect MetaMask Wallet](img/evm-sidechain-connect-metamask.png "Connect MetaMask wallet")
 
-### 2. Start the Transfer of Funds 
+### 2. Initiate the Transaction 
 
-Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount to send, and the destination address.
+Now that both Xumm and MetaMask wallets are connected to the bridge, you can select the direction of the bridge, the amount, and the address to fund.
 
-- **Direction of the bridge**: This is the direction of transfer; it can be either EVM sidechain → XRP Ledger Devnet or XRP Ledger Devnet → EVM sidechain. Use the “Switch Network” button to switch the direction of transfer.
-- **Amount to send**: This is the amount that you want to transfer to the other side. Note that there is a fee to use the bridge.
+- **Direction of the bridge**: The direction can be either EVM sidechain → XRP Ledger Devnet or XRP Ledger Devnet → EVM sidechain. Use the “Switch Network” button to switch the direction.
+- **Amount**: This is the transaction amount. Note that there is a fee to use the bridge.
```

Comment:
Maybe this should be **From amount**? I think that better matches the UI in the screenshot.

```suggestion
- **From Amount**: This is the transaction amount. Note that there is a fee to use the bridge.
```